woocommerce/woocommerce-android
# [Data Sprint] Track products orders and payments plugin once per site login per installation

Closes: #9588


### Description

The PR adds an event, that tracks:
* orders count
* products count
* payments gateways status
It's tracked once per the app installation per used site


More context - pdND4G-gg-p2#comment-243

### Testing instructions

* use logcat with "track" filtering
* run the app
* Notice the tracked event. e.g.
```
Tracked: application_store_snapshot, Properties: {"products_count":1078,"orders_count":778,"woocommerce-payments":"installed_and_activated","woocommerce-gateway-stripe":"not_installed","woocommerce-square":"installed_and_not_activated","woocommerce-paypal-payments":"installed_and_activated","blog_id":192152755,"is_wpcom_store":true,"was_ecommerce_trial":false,"plan_product_slug":"business-bundle","is_debug":true}
```

* Change site and close/open the app. Notice that it's tracked again for the new site
* Close/open without changing site - notice that it's not tracked
